lib/messages/status.js

```diff
@@ -20,28 +25,37 @@ class Status extends Message {
   }
 
   static getChecksPassAttachment(successfulChecks, totalChecks) {
-    let fallback;
+    let fallback, color;
     if (successfulChecks === totalChecks) {
       fallback = 'All checks have passed';
-    } else {
+      color = constants.STATUS_SUCCESS;
+    } else if (successfulChecks > 1) {
       fallback = `${successfulChecks} other checks have passed`;
-    }
+      color = constants.STATUS_PENDING;
+    } else if (successfulChecks === 1) {
+      fallback = `${successfulChecks} other check has passed`;
+      color = constants.STATUS_PENDING;
+    } else if (successfulChecks === 0) {
+      fallback = 'No checks have passed';
+      color = constants.STATUS_FAILURE
+    } 
+      
     return {
-      color: constants.STATUS_SUCCESS,
+      color,
       fallback,
       text: `:white_check_mark: ${fallback}`,
       footer: `${successfulChecks}/${totalChecks} successful checks`,
     };
   }
 
   renderAttachment() {
-    const summary = `${this.status.context}: ${this.status.description}`;
+    const summary = `${this.context}: ${this.description}`;
```

Comment:
I love this new way we're building Statuses for attachments on messages!

Just for due-diligence, I believe #866 could be fixed in this new format by adding a `null` check to the `description` field here. I didn't do deep testing yet, but I think it could be a quick `null` check to fix it!

Reply:
We're thinking the same thing, @IAmHughes! 👍